src/daos_vol_link.c

```diff
@@ -118,6 +118,7 @@ typedef struct H5_daos_link_write_ud_t {
     uint64_t max_corder;
     tse_task_t *link_write_task;
     tse_task_t *update_task;
+    tse_task_t *metatask;
```

Comment:
Does this new field get used? In H5_daos_link_create_task below, it looks like a dependency on a local metatask variable is registered, but it looks like this field's not used.

Reply:
you're right. i was experimenting with a fix for the link create issue and forgot to remove that.